backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/Importer.kt

```diff
@@ -10,6 +10,11 @@ object Importer {
         val logger = LoggerFactory.getLogger(Importer::class.java)
         val project = config.findProject()
 
+        if (project.pipelineName == null) {
+            logger.info("There is no store import pipeline for ${project.id}")
+            return true
+        }
+
         return try {
             logger.info("== Pipeline ${project.pipelineName} started ==")
             when (project.pipelineName) {
```

Comment:
Shouldn't Nuernberg be deleted in line 17? in the description of the pr you write "For the sozialpass projects we will use the store import module in the administration gui". So I expect this changes to apply to Nuernberg and Koblenz.